assets/js/shared/autocomplete.js

```diff
@@ -20,7 +23,10 @@ document.addEventListener('DOMContentLoaded', function () {
         const autocompleteSelect = document.getElementById(`${inputElement.id}-select`)
         const errorField = document.getElementById(`${inputElement.id}-error`)
 
-        if (!autocompleteInput.length) {
+        // Handling case where an autocomplete input box may be within a conditional radio button
```

Comment:
This allows us to use this one autocomplete.js for all our instances of autocomplete input, doing away with the need for separate versions per page.